src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -897,4 +902,34 @@ public void setSpectatorTarget(@Nullable org.spongepowered.api.entity.Entity ent
         return event;
     }
 
+    @Override
+    public WorldBorder getWorldBorder() {
+        if (this.worldBorder == null) {
+            this.worldBorder = cloneBorder((WorldBorder) this.world.worldBorder);
```

Comment:
The main reason why I do not default to the `World`'s world border is because if a plugin modifies the `Player`'s world border without having set it first, all of the players in the world would get the changes rather than just the specified player. Although, if similar behaviour occurs with scoreboards (which may not be relevant depending on how they are implemented in the game), then I will certainly change it.